sfe/overrides.go

```diff
@@ -556,7 +559,54 @@ type overrideRequest struct {
 // either the form logic is flawed or the requester has bypassed the form and
 // submitting (malformed) requests directly to this endpoint.
 func (sfe *SelfServiceFrontEndImpl) submitOverrideRequestHandler(w http.ResponseWriter, r *http.Request) {
-	// TODO(#8359): Check per-IP rate limits for this endpoint.
+	var refundLimits func()
+	var submissionSuccess bool
+	if sfe.limiter != nil && sfe.txnBuilder != nil {
```

Comment:
The addition of all this new (nested) code has made me realize how fragile this handler function is. It, like many of our WFE methods, has eight bajillion pairs of `http.Something` followed by an immediate `return`, and if any of those returns are missing the behavior will quickly go off the rails.

Not necessarily a thing to be fixed now, but a thing to be aware of for sure. This pattern is already bad in the WFE. Let's avoid perpetuating it in the SFE if we can.